MaterialLib/MPL/CreateProperty.cpp

```diff
@@ -35,7 +35,7 @@ std::unique_ptr<MaterialPropertyLib::Property> createProperty(
     using namespace MaterialPropertyLib;
     // Parsing the property type:
     //! \ogs_file_param{properties__property__type}
-    auto const property_type = config.getConfigParameter<std::string>("type");
+    auto const property_type = config.peekConfigParameter<std::string>("type");
```

Comment:
This is the correct statement. The check should happen in the specific type create method. That's probably the reason for the most of the failed tests....

Reply:
this is addressed in a seperate pull request